cdhweb/pages/migration_utils.py

```diff
@@ -0,0 +1,154 @@
+import json
+import logging
+from datetime import datetime
+
+from django.core.serializers import serialize
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+from django.test import TestCase
+
+logger = logging.getLogger('wagtail.core')
+
+# see for an explanation of django-treebeard & the `path` attribute:
+# http://www.agilosoftware.com/blog/django-treebard-and-wagtail-page-creation/
+def get_parent(apps, page):
```

Comment:
Why is this necessary? Don't pages have a `get_parent` method?

Reply:
It's not available in the context of a migration.